man/subscription-manager.8

```diff
@@ -594,7 +594,7 @@ Points to a certificate PEM file which contains the subscription certificate. Th
 .SS REDEEM OPTIONS
 The
 .B redeem
-command is deprecated, this command will be removed from the future major releases. This command is no-op, when simple content access mode is used. Command is used for systems that are purchased from third-party vendors that include a subscription. The redemption process essentially try to auto-attach the preselected subscription that the vendor supplied to the system.
+command is \fBdeprecated\fP and will be removed from the future major releases. This command is no-op, when simple content access mode is used. This command is used for systems that are purchased from third-party vendors that include a subscription. The redemption process essentially tries to auto-attach the preselected subscription that the vendor supplied to the system.
```

Comment:
"redeem" is unrelated to the access content mode; I see this was added (IMHO by mistake) in a previous commit that deprecated "redeem", however the explanation ought to be fixed: "redeem" is actually not a no-op.